test/codec-utils.test.js

```diff
@@ -0,0 +1,28 @@
+import Qunit from 'qunit';
+import { translateLegacyCodecs } from '../src/codec-utils';
+
+const { deepEqual, module, test } = Qunit;
```

Comment:
Well, I stand corrected, since it's already in one place, I don't have as strong a preference for one way or the other